### ISSUE TITLE
[Bug Accessibility] Properly label Buttons and check speech output

### ISSUE THREAD

**Issue by lenzone10** [palsoftware/pastiera#147]
Hi,
I am blind and use my Titan 2 using the Talkback screen reader. At the moment I am very happy with Pastiera which I find very useful and makes me write perfectly.
However, I found a flaw, when the keyboard is active, there are several buttons at the bottom of the screen that are not labeled with alternative text. I am referring to the buttons that are used to activate the various parts of the keyboard such as inserting emojis.
If the alternative texts are not present, Talkback simply says the word button when you touch it with a finger, without being able to understand what is activated.
Would it be possible to add labels to make this keyboard truly accessible? I hope I made myself understood correctly.
Thank's in advance.
Best regards.
Alessio

**Comment by pzauner** [comment]
Hi again Alessio,

I will personally take care of this soon.
If you have any further feedback, please feel free to share. I was also thinking about establishing a discussion page or dedicated accessibility feedback channel trying to make accessibility not an afterthought within this project.

Take care
Patrick

**Comment by lenzone10** [comment]
Hi Patrick,
Thank you for taking care of the problem promptly. You're right, accessibility is important, it's right to consider it.
I would like to try the updated version but unfortunately, I don't have a development environment to compile it and at the moment I don't think I will. Would it be possible to somehow have a compiled one to test?
Htank's again!
Best regards.
Alessio

**Comment by pzauner** [comment]
Yeah sure. We'll have to figure out how to build a nightly build channel I suppose. You may check this build:
https://github.com/palsoftware/pastiera/releases/tag/debug-build%2Fv.0.87

**Comment by lenzone10** [comment]
Hi Patrick,
I installed the build, now the keys are labeled but unfortunately a problem has been added.
Every time I enter a letter, the speech synthesis always repeats the type of keyboard layout and always reads all the buttons. In the previous version this did not happen.
Unfortunately, with this defect, it is not possible to write smoothly. See if you can do anything.
Thank's in advance.
Best regards.
Alessio

**Comment by pzauner** [comment]
yeah I know exactly what this causes. Will provide an update tomorrow
